config/locales/en.yml

```diff
@@ -1132,6 +1133,21 @@ en:
   ticket_column_unit_price: "Unit Price"
   ticket_column_total_price: "Total Price"
 
+  menu_1_title: "Shops"
+  menu_1_url: "/shops"
+  menu_2_title: "Map"
+  menu_2_url: "/map"
+  menu_3_title: "Producers"
+  menu_3_url: "/producers"
+  menu_4_title: "Connect"
+  menu_4_url: "https://openfoodnetwork.org/au/connect/"
+  menu_5_title: "Learn"
+  menu_5_url: "https://openfoodnetwork.org/au/learn/"
+  menu_6_title: "Blog"
+  menu_6_url: ""
+  menu_7_title: "Groups"
+  menu_7_url: ""
```

Comment:
I've just looked through again and I think Groups does have a default URL across all instances. It should be: `menu_7_url: "/groups"`.

Reply:
the footer link Groups is not related to this change. the footer still has the hard coded link Groups to /groups. This is just for the main menu where for example AUS, doesn't have Groups.

Reply:
We havent discussed the footer Navigate section that stays untouched with this PR: Shops /shops, Maps /map, Producers /producers, Groups /groups and About /<content_config_footer_about_url>.
We could do the same later with the Navigate footer links as we just did for the main menu.

Reply:
The "Groups" menu item does have a default route, unlike "Blog". Whether or not it's enabled is handled in the other file, no?